sky/serve/autoscalers.py

```diff
@@ -105,8 +128,10 @@ def __init__(self, spec: 'service_spec.SkyServiceSpec',
         super().__init__(spec)
         self.target_qps_per_replica: Optional[
             float] = spec.target_qps_per_replica
-        self.qps_window_size: int = qps_window_size
         self.request_timestamps: List[float] = []
+        self.num_overprovision: int = (
+            spec.num_overprovision if spec.num_overprovision is not None else
+            constants.AUTOSCALER_DEFAULT_NUM_OVERPROVISION)
```

Comment:
Do we really need overprovision in ondemand-only deployment? Could we move this var to SpotAutoscaler?

Reply:
Original reason for this is `OD+`.

Reply:
Possible that user might still want overprovisioning to tolerate cold start delay. (based on yesterday's Tyler's discussion)

Reply:
That makes sense! SG.